app/models/concerns/indexable.rb

```diff
@@ -107,6 +107,7 @@ def send_contact_export_message(data)
     # shoryuken_class is needed for the consumer to process the message
     # we use the AWS SQS client directly as there is no consumer in this app
     def send_message(body, options = {})
+      return nil if Rails.env.development?
```

Comment:
Why? We still need this to function on development right?

Reply:
we will revisit this in a different PR and issues